kyuubi-server/src/main/scala/org/apache/kyuubi/engine/EngineRef.scala

```diff
@@ -201,9 +202,10 @@ private[kyuubi] class EngineRef(
 
     MetricsSystem.tracing(_.incCount(ENGINE_TOTAL))
     try {
+      val started = System.currentTimeMillis()
+      conf.set(KYUUBI_ENGINE_SUBMIT_TIME_KEY, String.valueOf(started))
```

Comment:
I prefer move these two line to line 182

Reply:
> I prefer move these two line to line 182

done